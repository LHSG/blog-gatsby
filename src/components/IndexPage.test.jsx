import React from 'react';
import { shallow } from 'enzyme';
import IndexPage from './IndexPage';

import { imagePropType } from '../../test/fixtures';

test('rendering', () => {
  const wrapper = shallow(
    <IndexPage posts={[]} coverImageSizes={imagePropType} />,
  );
  expect(wrapper).toMatchSnapshot();
});
