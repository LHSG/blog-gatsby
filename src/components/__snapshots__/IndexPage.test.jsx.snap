// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<div>
  <Styled(Image)
    alt="Index page cover"
    sizes={
      Object {
        "sizes": "something",
      }
    }
    width="70vw"
  />
  <CoverDivider />
  <div />
</div>
`;
